crates/rspack_plugin_javascript/src/visitors/dependency/scanner.rs

```diff
@@ -99,7 +99,7 @@ impl DependencyScanner<'_> {
     use once_cell::sync::Lazy;
     use swc_core::common::comments::CommentKind;
     static WEBPACK_CHUNK_NAME_CAPTURE_RE: Lazy<regex::Regex> = Lazy::new(|| {
-      regex::Regex::new(r#"webpackChunkName\s*:\s*["|'|`]\s*(\w[\w0-9]*)\s*["|'|`]"#)
+      regex::Regex::new(r#"webpackChunkName\s*:\s*"(?P<_1>(\./)?([\w0-9_\-]+/)*?[\w0-9_\-]+)"|'(?P<_2>(\./)?([\w0-9_\-]+/)*?[\w0-9_\-]+)'|`(?P<_3>(\./)?([\w0-9_\-]+/)*?[\w0-9_\-]+)`"#)
```

Comment:
I couldn't visualize the regex. [Link](https://regexper.com/#webpackChunkName%5Cs*%3A%5Cs*%22%28%3FP%3C_1%3E%28%5C.%2F%29%3F%28%5B%5Cw0-9_%5C-%5D%2B%2F%29*%3F%5B%5Cw0-9_%5C-%5D%2B%29%22%7C'%28%3FP%3C_2%3E%28%5C.%2F%29%3F%28%5B%5Cw0-9_%5C-%5D%2B%2F%29*%3F%5B%5Cw0-9_%5C-%5D%2B%29'%7C%60%28%3FP%3C_3%3E%28%5C.%2F%29%3F%28%5B%5Cw0-9_%5C-%5D%2B%2F%29*%3F%5B%5Cw0-9_%5C-%5D%2B%29%60)

Is this regex the only solution? If it's possible, I hope you could write a regex that could be visualize.

Reply:
There are some special regex syntax that may be specific to rust (? P<name>expr)
https://docs.rs/regex/latest/regex/#grouping-and-flags

Below is a link to a visualization of this regular
https://regexper.com/#webpackChunkName%5Cs*%3A%5Cs*%28%22%28%28%5C.%5C%2F%29%3F%28%5B%5Cw0-9_%5C-%5D%2B%5C%2F%29*%3F%5B%5Cw0-9_%5C-%5D%2B%29%22%7C'%28%28%5C.%5C%2F%29%3F%28%5B%5Cw0-9_%5C-%5D%2B%5C%2F%29*%3F%5B%5Cw0-9_%5C-%5D%2B%29'%7C%60%28%28%5C.%5C%2F%29%3F%28%5B%5Cw0-9_%5C-%5D%2B%5C%2F%29*%3F%5B%5Cw0-9_%5C-%5D%2B%29%60%29

Reply:
Thanks. But I'm going to check out the usage and related tests of Webpack. I'm not sure we should support something(sub-folder) like `import(/* webpackChunkName: ./hello/elseo.js */, './foo.js')`.

Reply:
webpack uses this approach
```
// regexp to match at least one "magic comment"
const webpackCommentRegExp = new RegExp(/(^|\W)webpack[A-Z]{1,}[A-Za-z]{1,}:/);

	const comments = this.getComments(range);
		if (comments.length === 0) {
			return EMPTY_COMMENT_OPTIONS;
		}
		let options = {};
		/** @type {unknown[]} */
		let errors = [];
		for (const comment of comments) {
			const { value } = comment;
                        // check is magin comment
			if (value && webpackCommentRegExp.test(value)) {
				// try compile only if webpack options comment is present
				try {
					for (let [key, val] of Object.entries(
                                                   // value = webpackChunkName: "xxx"
						vm.runInNewContext(`(function(){return {${value}};})()`)
					)) {
						if (typeof val === "object" && val !== null) {
							if (val.constructor.name === "RegExp") val = new RegExp(val);
							else val = JSON.parse(JSON.stringify(val));
						}
						options[key] = val;
					}
				} catch (e) {
					const newErr = new Error(String(e.message));
					newErr.stack = String(e.stack);
					Object.assign(newErr, { comment });
					errors.push(newErr);
				}
			}
		}
		return { options, errors };
```

Reply:
> Thanks. But I'm going to check out the usage and related tests of Webpack. I'm not sure we should support something(sub-folder) like `import(/* webpackChunkName: ./hello/elseo.js */, './foo.js')`.

webpack doesn't support this so I don't think we need to either